scripts/git/git-merge-forward.sh

```diff
@@ -94,19 +94,19 @@ TOR_WKT_NAME=${TOR_WKT_NAME:-"tor-wkt"}
 MAINT_035_TB=( "maint-0.3.5" "" "$GIT_PATH/$TOR_WKT_NAME/maint-0.3.5" \
     "_035" "")
 # Used in maint/release merge and test branch modes
-MAINT_040=( "maint-0.4.0" "maint-0.3.5" "$GIT_PATH/$TOR_WKT_NAME/maint-0.4.0" \
-    "_040" "_035")
-MAINT_041=( "maint-0.4.1" "maint-0.4.0" "$GIT_PATH/$TOR_WKT_NAME/maint-0.4.1" \
+MAINT_041=( "maint-0.4.1" "maint-0.3.5" "$GIT_PATH/$TOR_WKT_NAME/maint-0.4.1" \
     "_041" "_040")
```

Comment:
Test branch mode needs to merge forward from 0.3.5 to 0.4.1 now.

Maybe there's some way to automatically derive these strings?
Or these tables?
We seem to make mistakes every time we do an update.

```suggestion
    "_041" "_035")
```